client/state/comments/reducer.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+import Immutable from 'immutable';
+
+/**
+ * Internal dependencies
+ */
+import {
+	COMMENTS_RECEIVE,
+	COMMENTS_REMOVE_COMMENT,
+	COMMENTS_ERROR_COMMENT,
+	COMMENTS_COUNT_RECEIVE,
+	COMMENTS_REQUEST,
+	COMMENTS_REQUEST_SUCCESS,
+	COMMENTS_REQUEST_FAILURE
```

Comment:
It'd be a great help if SERIALIZE and DESERIALIZE is implemented in this PR. We can have the comments subtree return default initial state for both until we're ready to persist this.